projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -48,6 +47,13 @@ export const withAIAssistant = createHigherOrderComponent(
 
 		const requestSuggestion = useCallback(
 			( promptType: PromptTypeProp, options: AiAssistantDropdownOnChangeOptionsArgProps ) => {
+				const clientIds = select( 'core/block-editor' ).getSelectedBlockClientIds();
```

Comment:
Ideally you should use `useSelect`: `const { getSelectedBlockClientIds } = useSelect( 'core/block-editor' );